data-sources.md

```diff
@@ -95,4 +95,6 @@ Quattroshapes used to be supported by Mapzen Search and its use was discontinued
 
 It has been replaced by Who's on First, which continues to provide global administrative place data (countries, regions, counties, cities) and administrative lookup (_"what country, region, and city is this address part of?"_).
 
+To help make the transition seamless, any queries that specify quattroshapes in the `sources` parameter will see results from Who's on First instead. Who's on First contains all the data from Quattroshapes, plus more data, and has continuous updates and fixes, so it should be a seamless transition where all existing queries will continue to work as expected.
```

Comment:
Good addition!

How about the second sentence be something like: Who's on First contains all the data from Quattroshapes, plus more data, and has continuous updates and fixes. All existing queries for Quattroshapes will continue to work without modification.

(Just to avoid using seamless again...and "without modification" because you think you're going to get qs data.)